src/app/inventory/store/deepsight.ts

```diff
@@ -33,3 +33,13 @@ export function isDeepsightResonanceSocket(socket: DimSocket): boolean {
     socketContainsPlugWithCategory(socket, PlugCategoryHashes.CraftingPlugsWeaponsModsMemories)
   );
 }
+
+export function isHarmonizable(item: DimItem) {
+  const isItemHarmonizable = item.sockets?.allSockets.filter(
```

Comment:
You can use `.some` to get a boolean out of this (and stop scanning when you find the right socket)

```suggestion
  const isItemHarmonizable = item.sockets?.allSockets.some(
```